src/status_im/qr_scanner/core.cljs

```diff
@@ -21,11 +22,16 @@
 
 (fx/defn set-qr-code
   [{:keys [db]} context data]
-  (merge {:db (-> db
-                  (update :qr-codes dissoc context)
-                  (dissoc :current-qr-context))}
-         (when-let [qr-codes (get-in db [:qr-codes context])]
-           {:dispatch [(:handler qr-codes) context data (dissoc qr-codes :handler)]})))
+  (let [[navigation-stack'] `({:chat-stack :home})]
```

Comment:
On the left side of the let binding, you have the assignment, which can use destructuring, which means you can assign only parts of the structure on the right to symbols on the left.

In this particular case you have on the right a list ``()` that contains a map {:chat-stack :home} and on the left part of the assignement you destructure that list to only name the first element `navigation-stack'`.
This is unecessary so you can just do 

```suggestion
  (let [navigation-stack {:chat-stack :home}]
```
note that I also remove the `'` at the end, which means nothing, we used to use it sometimes to avoid overwriting a name in the scope but it's a better practice to find another name instead. In that case there is no other `navigation-stack` symbol in the scope.